Packs/CommonPlaybooks/ReleaseNotes/2_3_99.md

```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Detonate and Analyze File - Generic
+
+The sub-playbook "Detonate and Analyze File - JoeSecurity" is no longer in use and has been replaced by the sub-playbook "Detonate File - JoeSecurity V2".
+
+##### Detonate File - Generic
+
+Added the sub playbook Detonate File - ThreatGrid v2 to replace the Deprecated and EOL sub playbook Detonate File - ThreatGrid. The playbook Detonate File - ThreatGrid will reach its end of life (EOL) at: Sep 01, 2023 and it will still be available for six months after the EOL date, Please use the integration ThreatGrid v2 instead.
```

Comment:
```suggestion
Added the Detonate File - ThreatGrid v2 sub-playbook to replace the deprecated and EOL Detonate File - ThreatGrid sub-playbook. The Detonate File - ThreatGrid sub-playbook will reach its end of life (EOL) at: Sep 01, 2023 and it will still be available for six months after the EOL date. Use the ThreatGrid v2 integration  instead.
```
Is the last sentence correct?  Should it be:
 Use the Detonate File - ThreatGrid v2 sub-playbook  instead.